skrub/_reporting/_data/templates/column-summaries.css

```
@@ -4,9 +4,11 @@
 /* Collection of column summary cards (as shown in the "column summaries" tab) */
 /* --------------------------------------------------------------------------- */
 
+/* The cards next to each other and horizontal scrolling. */
 .column-summary-group {
     display: flex;
-    flex-direction: column;
+    flex-direction: row;
+    flex-wrap: nowrap;
     gap: var(--spacing-l);
     min-width: min-content;
     max-width: max-content;
```

```suggestion
    width: max-content;
```
(note this will make it use less vertical space)